worker/provisioner/lxd-broker.go

```diff
@@ -54,13 +54,9 @@ func (broker *lxdBroker) StartInstance(args environs.StartInstanceParams) (*envi
 		return nil, errors.New("starting lxd containers with networks is not supported yet")
 	}
 	machineId := args.InstanceConfig.MachineId
-	bridgeDevice := broker.agentConfig.Value(agent.LxcBridge)
+	bridgeDevice := broker.agentConfig.Value(agent.LxdBridge)
```

Comment:
FWIW, as discussed on IRC - agent.LxcBridge is always empty, as it's not getting populated by providercommon.Bootstrap() used by most providers. So the `if` below is always going to succeed.